ts/src/profilers/profile-serializer.ts

```diff
@@ -81,7 +82,7 @@ class StringTable {
 function serialize<T extends ProfileNode>(
     profile: perftools.profiles.IProfile, root: T,
     appendToSamples: AppendEntryToSamples<T>, stringTable: StringTable,
-    ignoreSamplesPath?: string) {
+    hasLines: boolean, ignoreSamplesPath?: string) {
```

Comment:
nit: add `hasLines` to the jsdoc comment above as well.

Reply:
Done.